executor/common.h

```diff
@@ -469,6 +469,11 @@ static void loop()
 	// Tell parent that we are ready to serve.
 	reply_handshake();
 #endif
+#if GOOS_linux
+#if SYZ_EXECUTOR || SYZ_RESET_NET_NAMESPACE
+	checkpoint_net_namespace();
```

Comment:
This should go into the linux setup_loop() above. As you can see this code is already total mess as it's reused for C code generation, but if we start sprinkling it with each OS details, it will become unmanageable. These setup/reset_loop/test are specifically for per-OS code. There is a mention of akaros in the loop below but that's only because what happens on akaros is very weird (it does not have Go support atm).